scripts/jest/fiber.config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "modulePathIgnorePatterns": [
+    "/.module-cache/",
+    "<rootDir>/build/",
+    "<rootDir>/scripts/rollup/shims/",
+    "<rootDir>/scripts/bench/"
+  ],
+  "rootDir": "../../",
+  "transform": {
+    ".*": "./scripts/jest/preprocessor.js"
+  },
+  "setupFiles": [
+    "./scripts/jest/fiber.setup.js",
+    "./scripts/jest/environment.js"
+  ],
+  "setupTestFrameworkScriptFile": "./scripts/jest/test-framework-setup.js",
+  "testRegex": "/__tests__/.*(\\.js|coffee|ts)$",
+  "moduleFileExtensions": [
+    "js",
+    "json",
+    "node",
+    "coffee",
+    "ts"
+  ],
+  "roots": [
+    "<rootDir>/eslint-rules",
+    "<rootDir>/mocks",
+    "<rootDir>/scripts",
+    "<rootDir>/src",
+    "node_modules/fbjs"
+  ],
+  "collectCoverageFrom": [
```

Comment:
i think this one should go to package.json, since it's shared between both projects. (both projects share reporters, including `CoverageReporter` that uses this information)